.github/workflows/integration_test.yaml

```diff
@@ -0,0 +1,27 @@
+name: Integration tests
+
+on:
+  pull_request:
+
+jobs:
```

Comment:
Why not use operator workflows? According to team standards, we should use it for every repo we own: https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#ci-cd

Reply:
The integration test works for a charm but this is rather a application so it doesn't quite go well together unfortunately.